assets/css/privacy.scss

```diff
@@ -0,0 +1,66 @@
+/**
+ * privacy.scss
+ * Styles applied to the Privacy Policy Guide to support WooCommerce content.
+ * Adds support for styling ul/ol elements.
+ */
+
+/**
+ * Styling begins
+ */
+
+// Support for indented bullet-lists.
+.wp-suggested-text ul,
+.policy-text ul {
+	list-style: disc;
+}
+
+.wp-suggested-text ol,
+.policy-text ol {
+	list-style: decimal;
+}
+
+.wp-suggested-text ul li,
+.wp-suggested-text ol li,
+.policy-text ul li,
+.policy-text ol li {
+	margin-left: 2em;
+}
+
+// Pre-5.4 support for italics.
+.branch-4-9 .wp-suggested-text ul,
```

Comment:
Please remove support for 4.9.
This will probably get merged into 4.2.0, so you can also drop support for WP 5.0 and 5.1.
Sorry for ask to include support for it early, but we just dropped support for legacy versions: https://github.com/woocommerce/woocommerce/pull/26094

Reply:
Thanks @claudiosanches I've cleaned up the legacy support as well as made a minor update after review which was to switch from supporting both `.wp-suggested-text` and `.policy-text` to just using policy-text as it's always the present wrapper and wp-suggested-text is only really used with the core suggested text.